packages/document/main-doc/docs/en/blog/index.md

```diff
@@ -4,5 +4,70 @@ sidebar_position: 1
 
 # Blog
 
-You can find the latest news about Modern.js here.
+Find the latest update of modern.js and our thought processes here. Occasionally we explain the infrastructure behind it.
```

Comment:
It is suggested to capitalize the first letter of Modern.js, as it is a proper noun.

```suggestion
Find the latest update of Modern.js and our thought processes here. Occasionally we explain the infrastructure behind it.
```